appveyor.yml

```diff
@@ -0,0 +1,12 @@
+environment:
+  matrix:
+    - PYTHON: "C:\\Python37"
```

Comment:
Is this windows specific?

Reply:
Appveyor is for windows and Travis for unix